model/task/task.go

```diff
@@ -172,6 +172,8 @@ type Task struct {
 	// S3 when it is complete.
 	// TODO (EVG-7817): allow different statuses
 	ShouldSync bool `bson:"should_sync,omitempty" json:"should_sync,omitempty"`
+	// RunsSync indicates whether this task syncs its task directory to S3.
+	RunsSync bool `bson:"runs_sync,omitempty" json:"runs_sync,omitempty"`
```

Comment:
What's the difference between these 2 flags?

Reply:
`ShouldSync` indicates that a patch should run task sync at the end of the task. `RunsSync` is just a field that's set if the task runs task sync because either `ShouldSync` is true or the project config contains an s3.push command. I should probably rename `ShouldSync`, but there's already a TODO to change above it to change it, so I was just going to rename it then.